test/endpoints/workerStatus.test.js

```diff
@@ -0,0 +1,49 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import app from '../../server';
+import client from '../../server/helpers/redis';
+
+chai.use(chaiHttp);
+const fakeStatus = ['"2019-01-09T09:59:53.045Z"', 'Successfull', '2'];
+describe('Tests for automation endpoints\n', () => {
```

Comment:
Typo here:
...Tests for ~automation~ worker status endpoints\n', ...

Reply:
Corrected!